### PR TITLE
Gdr 2736

### PR DESCRIPTION
**Pull request by darsoo**
# Description
## What changed?
Related JIRA issue: 

## Why was it changed?

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [ ] Package version bumped
- [ ] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## gDRutils 1.3.17 - 2024-10-24
+*make average_biological_replicates_dt more restrictive - less column to check for duplicates
```
Comment (bczech) [inline NEWS.md:2]:
```suggestion
* make average_biological_replicates_dt more restrictive - fewer columns to check for duplicates
```

Comment (bczech) [reply NEWS.md:2]: more or less restrictive? 🤔

**Review by bczech — APPROVED** [review]

**Review by gladkia — APPROVED** [review]
GJ!

File: R/headers_list.R [hunk 2]
```diff
@@ -204,6 +204,11 @@
       "Fit Type GR",
       "RV_fit_type",
       "GR_fit_type"
+   ),
+   blacklisted = c(
```
Comment (gladkia) [inline R/headers_list.R:208]: Add unprettified versions as well

Comment (gladkia) [reply R/headers_list.R:208]: (see `fit_type` entries)

**Pull request merged**